run_fal.py

```diff
@@ -41,6 +41,8 @@
 parser.add_argument("--team-dist", action="store_true")
 parser.add_argument("--team-score", action="store_true")
 parser.add_argument("--anime-weekly-stats", action="store_true")
+parser.add_argument("--fansubs-file", default="fansubs05.txt")
```

Comment:
I don't believe we're awarding points for fansubs anymore?

Reply:
The rules say:

>+2000 points for each English-speaking region that simulcasts the anime in English in each of the following regions: USA, Canada, UK/Ireland, and Australia/New Zealand. The regions must be confirmed on MAL News' official Fall 2019 Simulcast List thread. The number of companies that simulcast an anime is irrelevant; only which region(s) it gets simulcast in matter. Simulcast points are counted in week 5.

The fansubs file with `simul simul simul simul` is used to check for that.

Reply:
i see. can we at least rename it to licenses.txt and license-file? no reason to use a legacy naming convention imo.

Reply:
We also have licenses.txt, the files are different. I guess we could call it simulcast.txt